curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.english.md

```diff
@@ -13,8 +13,9 @@ In ES5, we usually define a constructor function, and use the <code>new</code> k
 The class syntax simply replaces the constructor function creation:
 <blockquote>class SpaceShuttle {<br>&nbsp;&nbsp;constructor(targetPlanet){<br>&nbsp;&nbsp;&nbsp;&nbsp;this.targetPlanet = targetPlanet;<br>&nbsp;&nbsp;}<br>}<br>const zeus = new SpaceShuttle('Jupiter');</blockquote>
 Notice that the <code>class</code> keyword declares a new function, and a constructor was added, which would be invoked when <code>new</code> is called - to create a new object.<br>
-<strong>Note</strong><br>
```

Comment:
Per discussions on issue https://github.com/freeCodeCamp/freeCodeCamp/issues/35464, this should be:
```suggestion
<strong>Notes:</strong><br>
```